### DISCUSSION TITLE
Auto import music library

### DISCUSSION THREAD

**Discussion by ulrikstoch** [ciderapp/Cider#734] [Feature Request]
Maybe I'm missing something, but would love to use this as iTunes replacement without having a subscription to Apple Music. 

So it basically just imports my local library and I'm able to listen from Cider instead of iTunes.

**Comment by LawnT** [comment]
I second this! I would love to replace iTunes with Cider but there's not a really good way to import a massive local library without the app crashing. I think iTunes generates a library XML so maybe Cider parses that?
